README.md

```diff
@@ -356,7 +356,20 @@ Destroying models may result in errors due to foreign key integrity. First delet
 
 ## Running tests
 
-The tests in this repository are mainly integration tests, meaning you will need to run them using our preconfigured test server.
+### Own instance
+If you have a local or remote MySQL instance and would like to use that to run the test suite, use the following command:
+```bash
+MYSQL_HOST=<HOST> MYSQL_USER=<USER> MYSQL_PASSWORD=<PASSWORD> MYSQL_PORT=<PORT> CI=true npm test
```

Comment:
this won't work on Windows.  Maybe we specify this is only for Mac and Linux?  Better yet, specify what it should be on Windows..
```
set MYSQL_HOST=<HOST>
set MYSQL_USER=<USER> 
set MYSQL_PASSWORD=<PASSWORD> 
set MYSQL_PORT=<PORT> 
CI=true npm test
```